CHANGELOG.md

```diff
@@ -1,4 +1,10 @@
 # CHANGELOG
+# 1.0.0
+## May 22th, 2024
+
+- Add support command
```

Comment:
Can you pls teach the user how they'd use the new command and what it's for in this changelog?

i.e. "Added a support command, "Pieces: Get Support", to access our public support GitHub repository or connect with the Pieces team."